themes/bootstrap3/templates/Recommend/OpenLibrarySubjectsDeferred.phtml

```diff
@@ -1,9 +0,0 @@
-<?php
```

Comment:
The associated class for this module does not extend from AbstractSearchObjectDeferred, so trying to use this recommendation module does not work. Can you look into it? If nothing else, you could probably just restore this template as `<?=$this->render('Recommend/AbstractSearchObjectDeferred.phtml')?>`... but if the class inheritance chain can be easily cleaned up, that might be even cleaner.

Reply:
Thanks for the catch.  I don't want to change the module's inheritance (at least not in this PR) because it's far more complicated than the actual, trivial ASO subclasses.  Rendering the ASOD template is a little misleading as well for the same reason but I took care of that with a code comment.

Reply:
Looks good to me!